test/unit-test/scripts/npm/prepublish/lib/git-spec.js

```diff
@@ -4,113 +4,120 @@ process.env = {
+    execStub.mockRestore();
   } );
 
   it( 'should checkout the `master` branch', () => {
     util.git.checkoutMaster();
 
-    expect( execStub ).to.be.calledWith( 'git checkout master' );
+    expect( execStub ).toBeCalledWith( 'git checkout master' );
   } );
 } );
 
 describe( 'checkBranch', () => {
   beforeEach( () => {
-    execStub = sinon.stub( childProcess, 'exec' );
+    childProcess.exec = jest.fn();
+    execStub = jest.spyOn( childProcess, 'exec' );
   } );
 
   afterEach( () => {
-    execStub.restore();
+    execStub.mockRestore();
   } );
 
   it( 'should create a commit message using the passed version', () => {
     util.git.checkBranch();
 
-    expect( execStub ).to.be.calledWith( 'git rev-parse --abbrev-ref HEAD' );
+    expect( execStub ).toBeCalledWith( 'git rev-parse --abbrev-ref HEAD' );
   } );
 } );
 
 describe( 'commit', () => {
   beforeEach( () => {
-    execStub = sinon.stub( childProcess, 'exec' );
+    childProcess.exec = jest.fn();
+    execStub = jest.spyOn( childProcess, 'exec' );
   } );
 
   afterEach( () => {
-    execStub.restore();
+    execStub.mockRestore();
   } );
 
   it( 'should create a commit message using the passed version', () => {
     util.git.commit( '1.0.0' );
 
-    expect( execStub ).to.be.calledWith( 'git commit -am "1.0.0"' );
+    expect( execStub ).toBeCalledWith( 'git commit -am "1.0.0"' );
   } );
 
   it( 'should create a default commit message if no version is passed', () => {
     util.git.commit();
 
-    expect( execStub ).to.be.calledWith(
+    expect( execStub ).toBeCalledWith(
       'git commit -am "Auto-incrementing version"'
     );
   } );
 } );
 
 describe( 'tag', () => {
   beforeEach( () => {
-    execStub = sinon.stub( childProcess, 'exec' );
+    childProcess.exec = jest.fn();
+    execStub = jest.spyOn( childProcess, 'exec' );
   } );
 
   afterEach( () => {
-    execStub.restore();
+    execStub.mockRestore();
   } );
 
   it( 'should tag the corresponding commit with the passed version', () => {
     util.git.tag( '1.0.0' );
 
-    expect( execStub ).to.be.calledWith( 'git tag -a 1.0.0 -m "1.0.0"' );
+    expect( execStub ).toBeCalledWith( 'git tag -a 1.0.0 -m "1.0.0"' );
   } );
 } );
 
 describe( 'push', () => {
   beforeEach( () => {
-    execStub = sinon.stub( childProcess, 'exec' );
+    childProcess.exec = jest.fn();
+    execStub = jest.spyOn( childProcess, 'exec' );
   } );
 
   afterEach( () => {
-    execStub.restore();
+    execStub.mockRestore();
   } );
 
   it( 'pushes the commits and tags to the remote branch', () => {
     const repoURL = 'git+ssh://git@github.com/cfpb/capital-framework.git';
     return util.git.push( repoURL ).then( () => {
-      expect( execStub.firstCall ).to.be.calledWithExactly(
+      expect( execStub ).toHaveBeenCalledTimes( 3 );
+
+      /*
+      TODO:
+      The jest mocked function doesn't have a "firstCall", "secondCall", etc.
+      so we have to just check how many times exec has been called versus
```

Comment:
I think you can, the docs say each call is an array within the `calls` object and each parameter is a child of that. I'm pretty sure it'd look like this:

```
expect( execStub.calls[0][0] ).toBe( 'git push git+ssh://git@github.com/cfpb/capital-framework.git master' );
expect( execStub.calls[1][0] ).toBe( 'git push git+ssh://git@github.com/cfpb/capital-framework.git master:canary' );
etc...
```

From the second code block in https://facebook.github.io/jest/docs/en/mock-functions.html

Reply:
Oh hey look at that! Stackoverflow misled me! I'll try it out.

Reply:
Fixed! Thanks!

Reply:
💥